test/fixtures/change_requests.yml

```diff
@@ -1,17 +1,17 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
 one:
-  reqid: MyString
-  requestor_id: 
-  notebook_id: 
+  reqid: One
+  requestor_id: 1
+  notebook_id: 1
```

Comment:
Fixtures are created as the tests are loaded, as configured in `test/test_helper.rb`.
https://github.com/nbgallery/nbgallery/blob/cd6116dd11cf417d17d900aaf511a524d9329260/test/test_helper.rb#L8

Many fixtures had incorrect data that caused errors, so I tried to fix them.